### ISSUE TITLE
format issue of install_requires argument in setup.py

### ISSUE THREAD

**Issue by nuffin** [aidatatools/ollama-benchmark#10]
When I try to build with python 3.10.6, it issues

```shell
error in llm_benchmark setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after name and no valid version specifier)
    typer = 0.11.0
```

and I must remove spaces around `=` as below, then it works:

```python
    install_requires=[
        'click>=8.0.0',
        'typer==0.11.0',
        'ollama==0.1.8',
        'pyyaml==6.0.1',
        'requests==2.31.0',
        'psutil==5.9.8',
        'GPUtil==1.4.0',
        'lib-platform==1.2.10',
        'setuptools==69.1.0'
    ],
```

**Comment by chuangtc** [comment]
The bug was fixed in 0.3.21
https://pypi.org/project/llm-benchmark/0.3.21/
Please check it, and if the issue is resolved, please help close the ticket.
Thank you.

**Comment by nuffin** [comment]
confirmed.
